Sources/Host/Linux/PTrace.cpp

```diff
@@ -285,6 +285,16 @@ ErrorCode PTrace::getSigInfo(ProcessThreadId const &ptid, siginfo_t &si) {
   return kSuccess;
 }
 
+ErrorCode PTrace::getSpawnedTID(ProcessId pid, unsigned long &clonedTID) {
```

Comment:
This has the same issue as what we had before. The function name doesn't correspond to what the function does exactly.